### PR TITLE
Fixed audio/bag/binary feature tests to work on GPU

### PR DESCRIPTION
**Pull request by anmshkmr**

### CODE REVIEW

File: ludwig/encoders/bag_encoders.py [hunk 1]
```diff
@@ -64,8 +65,7 @@ def __init__(
             embeddings_on_cpu=embeddings_on_cpu,
             dropout=dropout,
             embedding_initializer=weights_initializer,
-        )
-
+        ).to(DEVICE)
```
Comment (ShreyaR) [inline ludwig/encoders/bag_encoders.py:68]: Not sure if you meant to leave this in.

Could you also remove the `DEVICE` initialization at line 28?

File: tests/ludwig/features/test_bag_feature.py [hunk 2]
```diff
@@ -33,6 +34,6 @@ def bag_config():
 def test_bag_input_feature(bag_config: Dict, encoder: str) -> None:
     bag_config.update({"encoder": encoder})
     bag_input_feature = BagInputFeature(bag_config)
```
Comment (ShreyaR) [inline tests/ludwig/features/test_bag_feature.py:36]: You might also need to move `bag_input_feature` to the correct device.

**Review by ShreyaR — APPROVED** [review]
Looks good! Added 2 minor comments -- feel free to merge once those are addressed.

File: ludwig/encoders/bag_encoders.py (hunk 1 above)
Comment (anmshkmr) [reply ludwig/encoders/bag_encoders.py:68]: Ah, thanks!

**Pull request merged**